src/etc/test-float-parse/Cargo.lock

```diff
@@ -2,28 +2,195 @@
 # It is not intended for manual editing.
 version = 3
 
+[[package]]
```

Comment:
Generally speaking tools are members of the global workspace, you probably want to add this one to the list: https://github.com/rust-lang/rust/blob/master/Cargo.toml#L3

That helps ensure that e.g. license checking is covered for their dependencies and they don't need special exceptions (e.g., https://github.com/rust-lang/rust/blob/fcaa6fdfbee1316184e7ad98c53241d52cd30a5f/src/tools/tidy/src/deps.rs#L68)

Reply:
Think this should be done.